### PR TITLE
[docs] Document proxy step

### PR DESCRIPTION
**Pull request by ngrudnitsky**

### CODE REVIEW

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 1]
```diff
@@ -1 +1,195 @@
-include::ROOT:page$under-construction.adoc[]
+= Plugin Web App
+
+== General Info
+
+The plugin provides functionality to ???.
```
Comment (ikalinin1) [inline docs/modules/plugins/pages/plugin-web-app.adoc:5]: to interact with Web applications?

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 2]
```diff
@@ -1 +1,195 @@
...
+
+== Installation
+
+.build.gradle
+[source,gradle]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-web-app', version: '0.2.0')
```
Comment (ikalinin1) [inline docs/modules/plugins/pages/plugin-web-app.adoc:12]: 0.2.4

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 3]
```diff
@@ -1 +1,195 @@
...
+
+== Properties
+
+=== Required
+
+There are no required properties for the plugin.
+
+=== Optional
+
+There are no required optional for the plugin.
```
Comment (ikalinin1) [inline docs/modules/plugins/pages/plugin-web-app.adoc:24]: mark it as a not implemented block, there are many optional properties

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 4]
```diff
@@ -1 +1,195 @@
...
+[source,gherkin]
+----
+Scenario: Verify step When I clear proxy log
+Given I am on a page with the URL 'http:/httpbin.org/get'
+When I clear proxy log
+Then number of HTTP GET requests with URL pattern `http://httpbin.org/get` is EQUAL TO `0`
+----
+
+
+=== Checks if the number of requests with given URL-pattern exist
```
Comment (ikalinin1) [inline docs/modules/plugins/pages/plugin-web-app.adoc:54]: Checks -> Check

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 5]
```diff
@@ -1 +1,195 @@
...
+When I clear proxy log
+Then number of HTTP GET requests with URL pattern `http://httpbin.org/get` is EQUAL TO `0`
+----
+
+
+=== Saves the query string from request with given URL-pattern into the variable with specified name and scopes.
+
+==== *_Info_*
+
+This step requires proxy to be turned on.
```
Comment (ikalinin1) [inline docs/modules/plugins/pages/plugin-web-app.adoc:94]: you could reuse the same block of the text using attributes https://docs.antora.org/antora/2.3/page/attributes/#attribute-entry

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 6]
```diff
@@ -1 +1,195 @@
...
+When I clear proxy log
+Then number of HTTP GET requests with URL pattern `http://httpbin.org/get` is EQUAL TO `0`
+----
+
+
+=== Checks if the number of requests with given URL-pattern exist
+
+==== *_Info_*
+
+This step requires proxy to be turned on. It can be done via setting properties or switching on @proxy metatag.
```
Comment (ikalinin1) [inline docs/modules/plugins/pages/plugin-web-app.adoc:58]: add property description into the optional properties?

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 7]
```diff
@@ -1 +1,195 @@
...
+[source,gherkin]
+----
+Scenario: Verify step When I clear proxy log
+Given I am on a page with the URL 'http:/httpbin.org/get'
+When I clear proxy log
+Then number of HTTP GET requests with URL pattern `http://httpbin.org/get` is EQUAL TO `0`
+----
+
+
+=== Saves the query string from request with given URL-pattern into the variable with specified name and scopes.
```
Comment (ikalinin1) [inline docs/modules/plugins/pages/plugin-web-app.adoc:90]: It is better to keep the titles short so they won't overflow right-hand TOC

**Comment by valfirst** [comment]
We plan to do full documentation review, so this PR will be merged as is to make sure we don't loose the important parts

**Pull request merged**